ITensorGPU/src/adapt.jl

```diff
@@ -7,12 +7,16 @@ function NDTensors.cu(eltype::Type{<:Number}, x)
 end
 NDTensors.cu(x) = fmap(x -> adapt(CuArray, x), x)
 
-function NDTensors.set_eltype_if_unspecified(
-  arraytype::Type{CuVector{T}}, eltype::Type
-) where {T}
+function NDTensors.specify_eltype(arraytype::Type{CuVector{T}}, eltype::Type) where {T}
```

Comment:
Did you mean to update this? I expected that we would be keeping `ITensorGPU` the same.

Reply:
This was a mistake, I thought this was NDTensorCUDA code. I have reverted the changes